webapp/src/main/java/life/qbic/views/login/resetPassword/LinkSentLayout.java

```diff
@@ -0,0 +1,35 @@
+package life.qbic.views.login.resetPassword;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import life.qbic.views.components.BoxLayout;
+
+/**
+ * <b> Defines the look of the password reset layout. </b>
+ *
+ * @since 1.0.0
+ */
+public class LinkSentLayout extends BoxLayout {
+
+  public Button loginButton;
+
+  public LinkSentLayout() {
+    fillLayoutComponents();
+  }
+
+  private void fillLayoutComponents(){
+    setTitleText("Email has been sent!");
+    setDescriptionText("Please check your inbox and click the received link to reset your password");
```

Comment:
NIT: "received link" sounds strange, Either "included" or maybe: "Please check your inbox and follow the instructions to reset your password."